index.html

```diff
@@ -24,9 +24,388 @@ <h1>Logo</h1>
+
+                            <div class="testimony--profile">
+                                <img src="" alt=""><span class="name">John Smith</span>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="slider-btn--wrapper">
+                        <button class="slider-btn slider-btn__right">
+                            <img src="" alt="">
+                        </button>
+                    </div>
+                </div>
+
+                <div>
+                    <h1>Let's grow together</h1>
+
+                    <button>Sign up</button>
+                </div>
+            </section>
+        </main>
+
+        <footer class="footer">
+            <div class="footer__subscribe--section">
+                <div class="newsletters--section">
+                    <h2>Always be in the loop.</h2>
+
+                    <p>Get news once a week, and don't worry — no spam.</p>
+                    
+                    <a href="#">Manage here</a>
+                </div>
+
+                <form action="" class="form">
+                    <input placeholder="Your email" type="text">
+                    <button>Subscribe</button>
+                </form>
+            </div>
+
+            <div class="footer__nav">
+                <a href="#">
+                    <span>Programming</span>
+                    <img src="" alt="">
+                </a>
+                
+                <a href="#">
+                    <span>DevOps</span>
+                    <img src="" alt="">
+                </a>
+                
+                <a href="">
+                    <span>Data Science</span>
+                    <img src="" alt="">
+                </a>
+                
+                <a href="">
+                    <span>Design</span>
+                    <img src="" alt="">
+                </a>
+            </div>
+
+            <div class="nav__links">
+                <a href="#">Articles</a>
+                <a href="#">About us</a>
+                <a href="#">Roadmaps</a>
+                <a href="#">We love Feedback</a>
+                <a href="#">Programming Tips</a>
+                <a href="#">Advertise/Partner</a>
+                <a href="#">Jobs</a>
+                <a href="#">Write for us</a>
+                <a href="#">Help center</a>
+                <a href="#">Privacy Policy</a>
+            </div>
+
+            <div class="follow-us">
+                <h4>Follow us</h4>
+
+                <div>
+                    <img src="" alt="" class="icons">
+                    <img src="" alt="" class="icons">
+                </div>
+            </div>
+        </footer>
```

Comment:
Overall Comment: 
Everything else is okay but I'll work on my own navigation bar because they're quite different